set.mm

```diff
@@ -15773,6 +15773,126 @@ Equality predicate (continued)
      potentially bind occurrences in "ph". $)
   $( $j free_var 'wsb' with 'y'; $)
 
+  ${
+    $d y x $.  $d y t $.  $d y ph $.
+    $( Alternate definition of proper substitution.  Note that the occurrences
+       of a given variable are either all bound ( ` x , y ` ) or all free
+       ( ` t ` ).  Also note that the definiens uses only primitive symbols.
+       It is obtained by applying twice Tarski's definition ~ sb6 which is
+       valid for disjoint variables, so we introduce a dummy variable ` y ` to
+       isolate ` x ` from ` t ` , as in ~ dfsb7 with respect to ~ sb5 .
+
+       This double level definition will make several proofs using it appear as
+       doubled.  Alternately, one could often first prove as a lemma the same
+       theorem with a disjoint variable condition on the substitute and the
+       substituted variables, and then prove the original theorem by applying
+       this lemma twice in a row.
+
+       This definition uses a dummy variable, but the justification theorem,
+       ~ sbjust , is provable from Tarski's FOL.
+
+       Once this is proved, more of the fundamental properties of proper
+       substitution will be provable from Tarski's FOL system, sometimes with
+       the help of specialization ~ sp , of the substitution axiom ~ ax-12 ,
+       and of commutation of quantifiers ~ ax-11 ; that is, ~ ax-13 will often
+       be avoided.  (Contributed by BJ, 22-Dec-2020.) $)
```

Comment:
"Once it is proved" refers to the definition like it was a theorem. Maybe rather "With this definition, ..."

Reply:
I think `this` was referring to `sbjust`, at which point there would be axioms mp thru 7.

I ended up deleting the paragraph though (https://github.com/metamath/set.mm/pull/3283/commits/d66989773f1fa16a5df1312b481040ba3db8aa5d)

Reply:
Indeed, that sentence was meant to be temporary, and deleted once the change has been made.